doc/source/themes/nature_with_gtoc/static/nature.css_t

```diff
@@ -321,3 +321,27 @@ div.seealso dd {
     margin-top: 0;
     margin-bottom: 0;
 }
+
+table {
```

Comment:
There is some styling for rendered_html tables above. Should that then be removed? (I think that was added for the converted styling notebook)

Also, would this also influence the tables in the other docs? So maybe this should have a 'dataframe' selector?